### DISCUSSION TITLE
Filter response time in query log (descending & ascending order)

### DISCUSSION THREAD

**Discussion by Gitonthegrid** [AdguardTeam/AdGuardHome#7310] [Ideas]
Sometimes I want to check where the high response time comming from, but I can't find a way/setting that let me filter descending and ascending order for the response time.

Wouldn't it be great to have that feature, so you can easily find the high response time requests?

**Comment by fvdm** [comment]
I would love this. With most queries appearing as cached with <1ms it is difficult to track down the slow responses that take many seconds to complete.

**Comment by krombel** [comment]
maybe a filter "Uncached responses" might help.

In the meantime I go the "hard" way to find slow queries:
```bash
grep -v '"Cached":true' /opt/AdGuardHome/querylog.json | jq '. | select(.Elapsed > 1000000000) | .QH' | sort | uniq -c
```
